azure_functions_worker/bindings/datumdef.py

```diff
@@ -92,6 +94,55 @@ def from_typed_data(cls, td: protos.TypedData):
 
         return cls(val, tt)
 
+    @classmethod
+    def from_rpc_shared_memory(cls, shmem: protos.RpcSharedMemory, shmem_mgr: SharedMemoryManager) -> Optional[Datum]:
```

Comment:
Hi @gohar94, from the discussion meeting, I think this method is for worker internal only. We're not exposing this to our customers, right? Just want to double confirm.

Reply:
Correct - this is just internal and used by the worker.
Any change you want me to make?

Reply:
No that's all good. Thanks